src/main/java/com/epam/preprod/sirenko/entity/Clothing.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.preprod.sirenko.entity;
+
+import java.util.Objects;
+
+/**
+ * Clothing entity.
+ *
+ * @author E.Sirenko
+ **/
+public class Clothing extends Products {
+	private String size;
```

Comment:
I think better approach for such field is to create enum with different sizes than use string type

Reply:
done